style.css

```diff
@@ -0,0 +1,197 @@
+html {
+  width: 100vw;
+}
+
+body {
+  background-color: #eceef1;
+  font-family: 'Open Sans', sans-serif;
+  margin: 0;
+}
+
+a {
+  color: #0077c5;
+  font-size: 12px;
+}
+
+/* header */
+
+header {
+  width: 100%;
+  padding-right: 5%;
+}
+
+/* footer */
+
+footer {
+  font-size: 12px;
+  color: rgb(54, 42, 42);
+}
+
+footer ul {
+  display: flex;
+  justify-content: center;
+  margin: 15px auto 0;
+}
+
+footer li {
+  list-style-type: none;
+  margin-right: 15px;
+}
+
+footer li a {
+  color: #4c4c4c;
+  text-decoration: none;
+  font-size: 11px;
+}
+
+footer p {
+  text-align: center;
+  color: #4c4c4c;
+  margin-top: 1px;
+}
+
+/* top buttons */
+
+#top-buttons {
+  float: right;
+  position: absolute;
+  right: 0;
+  display: flex;
+  align-items: center;
+  margin-right: 27px;
+}
+
+#top-buttons ul {
+  display: inline-block;
+}
+
+#top-buttons li {
+  list-style-type: none;
+  font-size: 12px;
+}
+
+#top-buttons button {
+  padding: 9px 20px;
+  margin-left: 6px;
+  display: flex;
+  align-items: center;
+}
+
+#signup-form img,
+#top-buttons button img {
+  width: 12px;
+  height: auto;
+  margin-right: 5px;
+}
+
+/* signup form */
+
+main {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+#logos {
+  margin: 12vh auto 0;
+  width: 350px;
+}
+
+#signup-form {
+  width: 27%;
+  height: 50%;
+  margin: 0 auto 20px;
+  background-color: #fff;
+  border: 1px solid #c7c7c7;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+#signup-form h2 {
+  color: #4c4c4c;
+  font-size: 1.2rem;
+}
+
+#signup-form p {
+  color: #4c4c4c;
+  font-size: 13px;
+  margin: 0 auto;
+  text-align: center;
+}
+
+#signup-form label {
+  color: #404040;
+  font-size: 14px;
+  line-height: 25px;
+}
+
+#signup-form label,
+#signup-form button {
+  margin-top: 30px;
+  width: 80%;
+}
+
+#signup-form input {
+  width: 100%;
+  height: 34px;
+  border: 1px solid #c1c5c8;
+  border-radius: 3px;
+  padding-left: 8px;
+}
+
+#signup-form button,
+#top-buttons button {
+  color: #fff;
+  background-color: #0077c5;
+  border: none;
+  border-radius: 4px;
+}
+
+#signup-form button img {
+  width: 14px !important;
+}
+
+#signup-form button {
+  padding: 9px 20px;
+  margin-left: 6px;
+  display: flex;
+  font-size: 16px;
+  align-items: center;
+  justify-content: center;
+}
+
+#privacy-statement {
+  font-style: italic;
+  font-size: 12px !important;
+  width: 67%;
+  padding: 40px 0 30px;
+  line-height: 1.9;
+}
+
+/* Align text centrally */
+
+#signup-form-captcha {
+  background-color: #eceef1;
+  width: 100%;
+}
+
+#signup-form-captcha p {
+  padding: 10px;
+}
+
+#signup-form-captcha a,
+#signup-form-captcha p {
+  font-size: 10px !important;
+}
+
+#privacy {
```

Comment:
Don't overuse ID selectors

Reply:
Hey Oscar, thanks for the great review, it has helped a lot and we've implemented the improvements you suggested.

We were discussing ID vs class selectors and I was wondering if you could help us better understand when ID selectors are appropriate and when class selectors are. Should we always go for class selectors first when possible?

Joe